sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/TaskConfigureAwaitTest.cs

```diff
@@ -34,12 +37,29 @@ public class TaskConfigureAwaitTest
         [TestCategory("Rule")]
         public void TaskConfigureAwait_NetFx() =>
             Verifier.VerifyAnalyzer(@"TestCases\TaskConfigureAwait.NetFx.cs", new TaskConfigureAwait());
+
 #else
+
         [TestMethod]
         [TestCategory("Rule")]
         public void TaskConfigureAwait_NetCore() =>
             Verifier.VerifyAnalyzer(@"TestCases\TaskConfigureAwait.NetCore.cs", new TaskConfigureAwait());
+
 #endif
 
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void TaskConfigureAwait_ConsoleApp()
```

Comment:
Why it's relevant for this analyzer to run as console app?

Reply:
This rule is relevant only for `OutputKind.DynamicallyLinkedLibrary` and there's a condition in ` OutputKind.DynamicallyLinkedLibrary` on line 46 for this (sorry, I can't get the link for some reason). This test covers that condition by using console app. Any other `OutputKind` would do the trick as well.